tempesta_fw/http.c

```diff
@@ -2216,8 +2300,9 @@ tfw_http_req_process(TfwConn *conn, struct sk_buff *skb, unsigned int off)
 				 */
 				TFW_WARN("Not enough memory to create"
 					 " a request sibling\n");
-				tfw_http_conn_msg_free((TfwHttpMsg *)req);
 				TFW_INC_STAT_BH(clnt.msgs_otherr);
+				tfw_http_drop(req, 500, "cannot create"
+					      " sibling request", true);
```

Comment:
Again, this seems like an internal error. Why the change to `tfw_http_drop()` Why the connection has to be closed in this case? Does the client need to know of this error in the browser?